Streetcode/Streetcode.BLL/MediatR/Transactions/TransactionLink/GetById/GetTransactLinkByIdHandler.cs

```diff
@@ -0,0 +1,32 @@
+﻿using AutoMapper;
+using FluentResults;
+using MediatR;
+using Streetcode.BLL.DTO.Transactions;
+using Streetcode.DAL.Repositories.Interfaces.Base;
+
+namespace Streetcode.BLL.MediatR.Transactions.TransactionLink.GetById;
+
+public class GetTransactLinkByIdHandler : IRequestHandler<GetTransactLinkByIdQuery, Result<TransactLinkDTO>>
+{
+    private readonly IMapper _mapper;
+    private readonly IRepositoryWrapper _repositoryWrapper;
+
+    public GetTransactLinkByIdHandler(IRepositoryWrapper repositoryWrapper, IMapper mapper)
+    {
+        _repositoryWrapper = repositoryWrapper;
+        _mapper = mapper;
+    }
+
+    public async Task<Result<TransactLinkDTO>> Handle(GetTransactLinkByIdQuery request, CancellationToken cancellationToken)
+    {
+        var transactLinks = await _repositoryWrapper.TransactLinksRepository.GetFirstOrDefaultAsync(f => f.Id == request.Id);
```

Comment:
Please space out the statement (lucky you StyleCop setup doesn't have max-line constraint).